client/src/pages/Home.jsx

```diff
@@ -38,7 +39,17 @@ export function Home() {
       rel="noopener noreferrer"
     >
       <FacebookIcon style={{ fontSize: 40, color: "black" }} />
+	 
     </IconButton>
+	<IconButton
+	component="a"
+	href="https://www.Twitter.com"
+	target="_blank"
+	rel="noopener noreferrer"
+	>
+	<TwitterIcon style={{ fontSize: 40, color: "blue" }} />
+	</IconButton>
+	
 	</Box>
 			</div>
 		<Footer />
```

Comment:
the formatting doesn't look right. Can we use eslint https://chariotsolutions.com/blog/post/configure-visual-studio-code-lint-react-jsx-syntax/

Reply:
Done